indico/modules/users/models/export.py

```diff
@@ -0,0 +1,133 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2023 CERN
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see the
+# LICENSE file for more details.
+
+import sqlalchemy as sa
+from sqlalchemy.dialects.postgresql import ARRAY
+
+from indico.core.db import db
+from indico.core.db.sqlalchemy import PyIntEnum, UTCDateTime
+from indico.util.date_time import now_utc
+from indico.util.enum import RichIntEnum
+from indico.util.i18n import L_
+
+
+class DataExportRequestState(RichIntEnum):
+    __titles__ = [L_('Pending'), L_('Running'), L_('Success'), L_('Failed'), L_('Expired')]
+    pending = 0  # Default value when there is no request
```

Comment:
I don't think `pending` is a good name for this, at least to me this implies you already submitted the request but it hasn't been picked up by the Celery worker yet. I'd say `none` would fit better.